examples/js/lines/LineSegments2.js

```diff
@@ -60,6 +60,83 @@ THREE.LineSegments2.prototype = Object.assign( Object.create( THREE.Mesh.prototy
 
 		return this;
 
-	}
+	},
+
+	raycast: ( function () {
+
+		var inverseMatrix = new THREE.Matrix4();
+		var ray = new THREE.Ray();
+		var sphere = new THREE.Sphere();
+
+		return function raycast( raycaster, intersects ) {
+
+			var precision = raycaster.linePrecision;
+
+			// If using a persepective-line material, there is a known precision:
+			// This doesn't work correctly though; the collision distance turns out too large.
+			// if(this.material && this.material.worldlinewidth) precision = this.material.worldlinewidth;
+
+			if(this.material.)
```

Comment:
This line does not look right...

Reply:
Oh, yeah, I forgot to update the js director as well as jsm. Code in two places is awkward!

Reply:
Also.. I don't see package-lock.json in my repo... I thought I got that right this time...?

Or.. you want me to add the origin package-lock.json?  I'll try that.